opentdf-dev.yaml

```
@@ -37,6 +37,9 @@ services:
     clientsecret: 'secret'
     realm: 'opentdf'
     legacykeycloak: true
+    unknownentities:
+      enabled: true
+      entitytypes: [emailAddress, userName, clientId]
```

You probably don't need `enabled`, right? Also, I'd suggest layering it a bit so you get better legibility and an extension point in case we need to add more detail somewhere.

```suggestion
    unknowns:
      entities:
        types: [emailAddress, userName, clientId]
```

---

I've changed this a bit.

---

New format:

```yaml
    inferid:
      from:
        clientid: false
        email: true
        username: true
```